plugins/chezmoi/README.md

```diff
@@ -0,0 +1,3 @@
+# Chezmoi plugin
+
+The chezmoi plugin defines a number of useful aliases for you. [Consult the complete list](./chezmoi.plugin.sh)
```

Comment:
Could you list the aliases in README (in a table like in some other plugins)? It shouldn't be difficult since there are only several aliases (unlike in `git.plugins.sh` where dozens of aliases are defined).

Reply:
Sure, will add it.

Reply:
Do you have a plan to do that near future?

Reply:
I got some bandwidth today, I'll add the aliases in table format.

Reply:
Added the aliases in the Readme.

Reply:
Thanks. Could you add a column of "Description"? Some other plugins have such a column, though there are a few that do not have it.

Reply:
Done

Reply:
Thanks!

Reply:
Some of the descriptions seem to be long. I've removed some descriptions of the detailed behavior in c42c8cbd0f19db4a5433b3d8d4febd736b2dbec5. Can you agree with this change?

Reply:
Yep, works!

Reply:
Great.